shopping_mall/backend/tests/agent/test_executor.py

```
@@ -0,0 +1,849 @@
+    return db
+
+
+def make_hitl_session(session_id=1, pending_action=None):
+    """ChatSession 목 — HitL 테스트 전용."""
+    s = MagicMock()
+    s.id = session_id
+    s.pending_action = pending_action
+    return s
+
+
+def make_hitl_exchange(id=42, user_id=10, status="pending_confirm"):
```

Ruff (0.15.9)

[error] Function argument `id` is shadowing a Python builtin

(A002)